cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -250,7 +250,7 @@ pScriptFor name Nothing help' =
 
 pScriptFor name (Just deprecated) help' =
       pScriptFor name Nothing help'
-  <|> ScriptFile <$> Opt.strOption
+  <|> File <$> Opt.strOption
```

Comment:
Why this change?

Reply:
Because `ScriptFile` is redundant

Reply:
But it was meant to disambiguate a file containing anything from a file containing a script right? And this distinction still makes sense I think.

Anyway it's a small thing, feel free to ignore and resolve.

Reply:
That's the purpose of a type parameter of `File`. `type ScriptFile = File ScriptInAnyLang In`.